src/azul/service/manifest_service.py

```diff
@@ -216,7 +215,8 @@ def get_manifest(self,
         generator = ManifestGenerator.for_format(format_=format_,
                                                  service=self,
                                                  catalog=catalog,
-                                                 filters=filters)
+                                                 filters=filters,
+                                                 file_url_func=file_url_func)
```

Comment:
Each generator has a reference to the service so we don't need to parameterize each generator.